README.md

```diff
@@ -242,6 +242,13 @@ const node = new IPFS({
     }
   }
 })
+
+// Events
+
+node.on('init', () => {})     // Node has successfully finished initing the repo
```

Comment:
We still need an event/callback for after `new IPFS({init: false, start: false})` as that can be async as well